settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@jforscht Please remove both "icon.png" and "settings.json" files from your pull request as they are not required in the open-source repository in GitHub.

Reply:
@jforscht please remove the icon and the settings.json files from your folder "Impower" on your local machine. then push your changes by following the commands below.
```
git add -A
git commit -m "Your message"
git push
```

If that doesn't work, you can close this PR and submit a new one with only the files below.

1. apiDefinition.swagger.json
2. apiProperties.json
3. readme.md

Thank you so much.